src/components/Chat/InputMessage.js

```diff
@@ -165,7 +165,7 @@ const styles = StyleSheet.create({
     right: 0,
     height: 25,
     width: 25,
-    backgroundColor: COLORS.holyTosca,
+    backgroundColor: colors.holytosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been changed. Ensure that the new color (`colors.holytosca`) is equivalent to the old one (`COLORS.holyTosca`) in terms of user experience and accessibility.
```diff
- backgroundColor: COLORS.holyTosca,
+ backgroundColor: colors.holytosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->